README.md

```diff
@@ -1,2 +1,37 @@
 # storcycle_api
-StorCycle API Examples
+
+[Swagger Documentation](https://localhost/apidocs/)
+
+## StorCycle API Examples
+
+This repository contains examples for interacting with the StorCycle API. These general examples require either creation of the example directory tree, or adjusting variables to suit the local system.
+
+To use the examples, client code must first be generated in the desired language.
+
+### Python:
+
+Download a client generator plugin for your local IDE and install. Note that while the OpenAPI Generator plugin by Jim Schubert was used to generate the example Python client, any client generator should be adequate.
```

Comment:
These instructions appear to be hard-coded to assuming an intellij IDE. Since this is for python, did you check that pycharm has the OpenAPI generator plugin?

We should probably include command line instructions as well and a link to the open api website for downloading their library directly.

Reply:
@RachelTucker, I have checked that PyCharm has the OpenAPI generator plugin.

I can add command line instructions. To what are you referring with downloading the open api website library directly? I'm not seeing anything on [https://www.openapis.org/](https://www.openapis.org/) for libraries.

Reply:
https://openapi-generator.tech/docs/installation/

Reply:
Awesome, I've already included that link, I just misunderstood this context. Thanks!